### ISSUE TITLE
How do we know which data reporting system a single agency participates in?

### ISSUE THREAD

**Issue by jeremiak** [18F/crime-data-api#495]
We want to get UCR participation data for a single agency so that we can know if it submits SRS or NIBRS. Right now there is a `/participation/agencies` endpoint but it returns all agencies at once. The implications of this for the front end are fetching _all_ the data and then filtering it down by agency ORI even if the user just wants to see a single agency.

I noticed some keys in the `/agencies/:ori` endpoint that seemed promising but both were `null`

```
  "months_reported": null,
  "nibrs_months_reported": null,
```

What is the best way to get this information given a single ORI?

**Comment by harrisj** [comment]
Hmm, I will have to look into why those were `null`. Do you have the specific API URL?

In response to your question though, it's complicated. Right now, we have `reported` and `nibrs_reported` because the database allows us to see if a given agency has filed a Return A report and/or a NIBRS report for any given month. The complication is that the Return A reports may either be from direct SRS reports or back-converted from NIBRS and that some agencies may submit both types of reports. In the future, there is a `source_type` field for the `reta_month` table that would indicate if it came from NIBRS or SRS, but it's just set to `R` for all reports from the mainframe (it'll have to wait until the new UCR workflow begins).

Okay, tl;dr, what does this mean? For a given month and agency, I can only tell if:
1. If an agency has submitted either SRS or NIBRS (`reported` and `months_reported`)
2. If an agency has submitted NIBRS (`nibrs_reported` and `nibrs_months_reported`)

But because some unknown number of agencies submit both types of reports and the source flag is not currently supported, you can't simply subtract agencies that have reported NIBRS from reporting agencies to get agencies that only submit SRS. Sorry...

**Comment by jeremiak** [comment]
The specific URL where those `null` values is `/agencies/NJ0110500`.

So, if we wanted to display the NIBRS table only for agencies that submit NIBRS we should check for a `months_reported` value? Or should we look for both `nibrs_reported` and `nibrs_months_reported`?

**Comment by harrisj** [comment]
This seems to be a duplicate of #501, which was fixed. I am going to close it.
